src/main/java/codeu/model/data/:q

```diff
@@ -0,0 +1,61 @@
+// Copyright 2017 Google Inc.
```

Comment:
I think you added this file by mistake! Please delete.

Reply:
Yeah. I don't know how I even created that. I was trying to use Vim for the first time. I will delete it.